prdoc/pr_7980.prdoc

```diff
@@ -0,0 +1,8 @@
+title: '`fatxpool`: optimize txs prunning based on inactive views provides tags '
+doc:
+- audience: [Node Operator, Node Dev]
+  description: |-
+   Optimize the transaction pruning on a new best block event, on a new enacted block. It removes transactions based on the provides tags present in the view store's inactive views if not found in the active view of this new best block that was just notified, instead of revalidating the transactions in the runtime, to obtain the provides tags, which is a costly operation.
```

Comment:
```suggestion
   Optimize the transaction pruning on a new best block event, on a new enacted block. It removes transactions based on the provides tags present in the view store's inactive views if not found in the active view of the new best block that was just notified, instead of revalidating the transactions in the runtime, to obtain the provides tags, which is a costly operation.
```